nflog/nflog.go

```diff
@@ -0,0 +1,585 @@
+// Copyright 2016 Prometheus Team
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// A package implementing a garbage-collected and snapshottable append-only log of
```

Comment:
Nit: Should start with `// Package nflog implements…`

(BTW: Perhaps this is a stupid question by a non-computer-scientist, but where is the "f" in nflog coming from?)